src/components/views/elements/MemberEventListSummary.js

```diff
@@ -107,10 +107,17 @@ module.exports = React.createClass({
                 </span>
             );
         }
+
+        // Return null instead of an empty <span> if there is no joinSummary
```

Comment:
If a user joins and leaves, it doesn't come up in the summary, but it will appear in 'joinAndLeft' and also the expanded event. The `joinEvents` and `leaveEvents` are representative of the net movement per-user.